client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,272 @@
 import React from 'react';
+import SmartphoneOutlinedIcon from '@mui/icons-material/SmartphoneOutlined';
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import TimerOutlinedIcon from '@mui/icons-material/TimerOutlined';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Box, Typography, Link as MUILink } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { styled } from '@mui/material/styles';
+import scrollToTop from '../../utils/scrollToTop/scrollToTop';
```

Comment:
please use the index.js file inside the utils folder. then you can just import your function in such way:

```suggestion
import scrollToTop from '../../utils';
```

Reply:
solved